lib/msf/core/modules/external/templates/multi_scanner.erb

```diff
@@ -27,6 +27,7 @@ class MetasploitModule < Msf::Auxiliary
 
   def run_batch(ips)
     datastore.delete('RHOSTS')
+    datastore.remove_option('RHOSTS') if self.datastore.is_a?(Msf::DataStoreWithFallbacks)
```

Comment:
Required because the original datastore had a `delete` method that also removed data options (Which had other bugs associated with it)

https://github.com/rapid7/metasploit-framework/pull/16940/files#diff-09925245579a8207d4933e78a8e76c5168fe2a1134f0b02a13f9ff344e2c8dc4